src/v/features/feature_table.h

```diff
@@ -181,6 +182,12 @@ constexpr static std::array feature_schema{
     feature::replication_factor_change,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{7},
```

Comment:
Not a comment for reviews but more for my understanding of cluster version. From the code base it seems `cluster version` indicates the type of content on raft0. My question is: What content does version number 7 refer to?

Reply:
It's a number that represents the feature set that's supported. We bump it when we've added new features, and make a release. (Well, we bump it at the first new feature on dev).

Reply:
@NyaliaLui i won't leave the link here, but there is an internal wiki page on the feature that you can find in confluence.